.eslintrc.json

```diff
@@ -5,5 +5,18 @@
     ],
     "rules": {
         "no-unused-vars": ["error", {"vars": "all", "args": "none"}]
-    }
+    },
+    "overrides": [
+        {
+            "files": [
+                "index.js",
+                "server.js",
+                "gulpfile.js",
+                "Source/Workers/transferTypedArrayTest.js"
+            ],
+            "parserOptions": {
+                "sourceType": "script"
```

Comment:
1. eslint's default is script, but `Tools/eslint-config-cesium/browser.js` is the standard Cesium ecosystem package for browser based development, which now sets `module` to the default. The top-level .eslintrc.json inherits from it.
2. We need to set it back here for `transferTypedArrayTest` because it is not a module, it doesn't get transpiled which is why it lives in Workers and not WorkersES6.  It is used only by the worker system to detect transferable typed array support.
3. cesiumWorkerBootstrapper.js is ignored completely by eslint in master as well, it contains an entire copy of requirejs, verbatim, used by the worker system so linting it isn't useful.